frontend/js/commento.js

```diff
@@ -1935,22 +1937,27 @@
     });
 
     if (configuredOauths["sso"]) {
-      var button = create("button");
+      if (autoSSO && numOauthConfigured === 0 && configuredOauths["commento"] === false){
+        global.commentoAuth({"provider": "sso", "id": id});
+        setTimeout(global.loginBoxClose, 250);
+      } else {
```

Comment:
This and the end closing brace are all that is necessary.  If nothing else have been configured there isn't any point in the additional argument, just auto log them in.

Reply:
```go
      if (numOauthConfigured === 0 && configuredOauths["commento"] === false) {
        global.commentoAuth({ "provider": "sso", "id": id });
        setTimeout(global.loginBoxClose, 250);
      } else {
        ......
      }
```